R/step-subset-select.R

```diff
@@ -61,7 +61,10 @@ dtplyr_tidyselect <- function(.data, ...,
     abort("The use of `where()` is not supported by dtplyr.", call = .call)
   }
   sim_data <- simulate_vars(.data, drop_groups = .drop_groups)
-  tidyselect::eval_select(expr(c(!!!dots)), sim_data, allow_rename = .allow_rename)
+  tidyselect::eval_select(expr(c(!!!dots)), sim_data,
```

Comment:
Minor formatting adjustment - can this be changed to one of these 2 options?

Up to you which one. I personally think Option 2 looks a bit odd and doesn't work well with RStudio's auto-indent but that pattern is used elsewhere in dtplyr.

``` r
# Option 1
tidyselect::eval_select(
  expr(c(!!!dots)),
  sim_data,
  allow_rename = .allow_rename,
  error_call = .call
)

# Option 2
tidyselect::eval_select(expr(c(!!!dots)),
  sim_data,
  allow_rename = .allow_rename,
  error_call = .call
)
```

Other than that this looks good to merge.